From f2e11f515dbbc840369c3d4a946d61366dc6dd9d Mon Sep 17 00:00:00 2001
From: Da Xin <da.xin@wsu.edu>
Date: Sun, 12 Apr 2020 20:07:37 -0700
Subject: [PATCH 2/2] issue 33428 Data links in Pandas API Reference are broken
 404

---
 pandas/plotting/_misc.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/pandas/plotting/_misc.py b/pandas/plotting/_misc.py
index 47a4fd8ff..e05c61c19 100644
--- a/pandas/plotting/_misc.py
+++ b/pandas/plotting/_misc.py
@@ -358,8 +358,7 @@ def parallel_coordinates(
     Examples
     --------
     >>> from matplotlib import pyplot as plt
-    >>> df = pd.read_csv('https://raw.github.com/pandas-dev/pandas/master'
-                        '/pandas/tests/data/csv/iris.csv')
+    >>> df = pd.read_csv('https://raw.github.com/pandas-dev/pandas/master/pandas/tests/data/iris.csv')
     >>> pd.plotting.parallel_coordinates(
             df, 'Name',
             color=('#556270', '#4ECDC4', '#C7F464'))
-- 
2.24.1.windows.2

